### PR TITLE
Upgrade appveyor to use Visual Studio 2017, issue #507

### PR DESCRIPTION
**Pull request by jwillemsen**
* appveyor.yml:

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -12,9 +12,9 @@ install:
   - git clone -q -b Latest_Micro --depth 1 git://github.com/DOCGroup/ACE_TAO.git
   - cd ..
   - set MPC_ROOT=%CD%\ext\MPC
-  - call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
+  - call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" amd64
   - call configure --no-tests --ace=ext\ACE_TAO\ACE
-  - ext\ACE_TAO\ACE\bin\mwc.pl -type vc14 tests\DCPS\Messenger
+  - call perl ext\ACE_TAO\ACE\bin\mwc.pl -type vs2017 tests\DCPS\Messenger
```
Comment (mitza-oci) [inline appveyor.yml]: Why is "call perl" needed here?  Did perl become a batch script somehow?

Comment (jwillemsen) [inline appveyor.yml]: Just removed that part, let us see if it works on appveyor

**Comment by jwillemsen** [comment]
Removing call worked, will make the similar change for ATCD

**Comment by jwillemsen** [comment]
Similar changes are made to ACE/TAO, CIAO, and DAnCE

**Pull request merged**